packages/manager/apps/web/client/app/hosting/general-informations/general-informations.routing.js

```diff
@@ -0,0 +1,9 @@
+import template from './GENERAL_INFORMATIONS.html';
+
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state('app.hosting.dashboard.information', {
+    url: '',
```

Comment:
empty url?

Reply:
Yes, this will correspond to /hosting/{serviceName} and will be the landing tab